client/my-sites/earn/home.tsx

```diff
@@ -121,18 +123,18 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		const title = hasConnectedAccount
 			? translate( 'Manage Recurring Payments' )
 			: translate( 'Collect recurring payments' );
-		const body = hasConnectedAccount
-			? translate(
-					"Manage your subscribers, or your current subscription options and review the total revenue that you've made from recurring payments."
-			  )
-			: translate(
-					'Charge for services, collect membership dues, or take recurring donations. Automate recurring payments, and use your site to earn reliable revenue. {{em}}Available to any site with a paid plan{{/em}}.',
-					{
-						components: {
-							em: <em />,
-						},
-					}
-			  );
+		const body = hasConnectedAccount ? (
+			translate(
+				"Manage your subscribers, or your current subscription options and review the total revenue that you've made from recurring payments."
+			)
+		) : (
+			<Fragment>
+				{ translate(
+					'Charge for services, collect membership dues, or take recurring donations. Automate recurring payments, and use your site to earn reliable revenue.'
+				) }{' '}
```

Comment:
Breaking up the text makes sense, but it does lead to this slightly messy way of adding a space. Should we put the emphasised text in its own paragraph? 

Could the translation of the two sentences be affected by context and that they are being used together? If so, perhaps we should keep the full version when a site doesn't match the plan requirements.